nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
 ---
-title: Microsoft Teams JavaScript client library
+title: Microsoft Teams Client API reference
 description: Reference documentation for the latest Microsoft Teams JavaScript client library
```

Comment:
Let's generalize this title to include all the libraries in the TOC. How about:

title: Microsoft Teams SDK reference
description: Reference documentation for Microsoft Teams SDK

(Using "SDK" in the holistic sense to include all the API libraries listed in the doc)